massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -336,11 +337,9 @@ impl ProtocolWorker {
         cmd: ProtocolCommand,
         timer: &mut std::pin::Pin<&mut Sleep>,
     ) -> Result<(), ProtocolError> {
-        massa_trace!("protocol.protocol_worker.process_command.begin", {
-            "cmd": cmd
-        });
         match cmd {
-            ProtocolCommand::IntegratedBlock { block_id } => {
+            ProtocolCommand::IntegratedBlock { block_id, storage } => {
+                // TODO properly manage storage
```

Comment:
You could refer to https://github.com/massalabs/massa/issues/2883

Reply:
I voluntarily didn't touch this to not collide with https://github.com/massalabs/massa/issues/2883 and https://github.com/massalabs/massa/pull/2888